imgur.sh

```diff
@@ -73,7 +73,7 @@ while [ $# -gt 0 ]; do
 	url="${url%%</link>*}"
 	delete_hash="${response##*<deletehash>}"
 	delete_hash="${delete_hash%%</deletehash>*}"
-	echo $url
+	echo $url | sed 's/^http/https/'
```

Comment:
If you just add a colon to the end of this I'll merge it; `s/^http:/https:/`. I know it likely doesn't ever make any practical difference but I like to be sure.